pkg/tecdsa/preparams.go

```diff
@@ -0,0 +1,15 @@
+package tecdsa
+
+import "github.com/binance-chain/tss-lib/ecdsa/keygen"
+
+// PreParams represents tECDSA DKG pre-parameters that were not yet consumed
+// by DKG protocol execution.
```

Comment:
Maybe just:
```suggestion
// PreParams represents a pre-parameters set that is required for tECDSA DKG participation.
```
I think there is no need to mention how those parameters are used in detail. We are on a quite high abstraction level here.

Reply:
I am not sure about it. I mentioned they are not yet consumed because they do not have `PrivateKey paillierSK`. This representation makes sense only pre-DKG and this sentence is to ensure no one tries to use them or thinks to associate them with DKG execution.